devito/dse/symbolics.py

```diff
@@ -169,30 +153,39 @@ class Rewriter(object):
     Transform expressions to reduce their operation count.
     """
 
+    """
+    Track what options trigger a given transformation.
+    """
     triggers = {
         '_cse': ('basic', 'advanced'),
         '_factorize': ('factorize', 'advanced'),
         '_optimize_trigonometry': ('approx-trigonometry', 'advanced'),
-        '_replace_time_invariants': ('glicm', 'advanced')
+        '_replace_time_invariants': ('glicm', 'advanced'),
+        '_split_expressions': ('split', 'devito3.0')  # TODO: -> 'advanced' upon release
```

Comment:
Because it'd otherwise break ``legacy`` for a very stupid reason  (I seem to remember something like missing declarations)